### ISSUE TITLE
Top issues before multi client testnet

### ISSUE THREAD

**Issue by terencechain** [prysmaticlabs/prysm#4781]
- [x] Attestation signature fails to verify for fork choice accounting #4773
- [x] Increasing # of users reporting OOM #4813
- [x] A lot of forking showed on the block tree visualizer. Concerning given it's a testnet with artificially short block time
- [x] Dynamic subscription of committee subnet
- [x] Attester should submit attestation right away if head slot == current slot
- [x] Discv5 Infra set up
- [x] Update BLS library to v0.10 and IETF standard #4804
- [x] Ensure local interop works, making it easier for clients to test running a chain from genesis with
 Prysm to ensure finality, participation, and heterogenous spread of validators across the clients
- [x] Add Support for [Noise](https://github.com/libp2p/specs/tree/master/noise)
- [x] Resolve profitability issues #4589
- [x] Update to spec v0.10.1
- [x] Update to spec v0.11
- [x] Remove `keystore` key manager
- [x] Add IPv6 support
- [x] A proper tool to migrate DB to be new state mgmt service compatible
- [x] Update to spec v0.11.1

**Comment by somuchid** [comment]
some validators are penalized while some are rewarded {{4589}}

**Comment by handelaar2** [comment]
ipv6 support?  (#3420)

**Comment by mcdee** [comment]
Any thoughts about removing the `keystore` keymanager?  Now seems a good a time as any to do so.

**Comment by prestonvanloon** [comment]
Yes. Let's remove `keystore` manager

**Comment by tzapu** [comment]
once this is done, will validators currently using the keystore manager just fail to start after a pull?
if so, would you consider adding a console message with migrations instructions? if you think it's worth it? (non issue if it will only take effect on the testnet restart i guess)

**Comment by nisdas** [comment]
@tzapu it will only happen in our testnet restart, yeah that sounds like a good idea. We can link the instructions in a log to our docs, so that users who are not sure can still be able to use the new key manager

**Comment by tzapu** [comment]
@nisdas if it s on testnet restart, then it's not gonna be so important, there will be a lot of other manual things to be done anywa... but the participate website should be updated :D

**Comment by rauljordan** [comment]
Closing in favor of #5227
